maldoca/ole/oss_utils.cc

```diff
@@ -66,33 +65,76 @@ inline void StripNullChar(std::string* str) {
 }  // namespace
 
 bool BufferToUtf8::Init(const char* encode_name) {
+#ifndef MALDOCA_CHROME
   if (converter_ != nullptr) {
     iconv_close(converter_);
   }
+#else
+  if (converter_to_unicode_ != nullptr) {
+    ucnv_close(converter_to_unicode_);
+  }
+  if (converter_to_utf8_ != nullptr) {
+    ucnv_close(converter_to_utf8_);
+  }
+#endif
   internal_converter_ = InternalConverter::kNone;
   // Fixing missing encoding;
   // cp10000 is calld MAC in iconv
   if (strcmp(encode_name, "cp10000") == 0) {
     encode_name = "MAC";
     DLOG(INFO) << "Replaced cp10000 with MAC";
   }
+#ifndef MALDOCA_CHROME
   converter_ = iconv_open("UTF-8", encode_name);
   if (converter_ == reinterpret_cast<iconv_t>(-1)) {
     converter_ = nullptr;
     LOG(ERROR) << "Fail to open iconv for " << encode_name << ": " << errno;
+#else
+  UErrorCode err_to_unicode;
+  UErrorCode err_to_utf8;
+  converter_to_unicode_ = ucnv_open(encode_name, &err_to_unicode);
+  converter_to_utf8_ = ucnv_open("UTF-8", &err_to_utf8);
+
```

Comment:
add a comment that either both succeed or both fail?

Reply:
what do u mean thereby? it can happen that one fails and one succeeds, e.g. if one passes an unknown encode_name.

Reply:
if say err_to_unicode succeeds but error_to_utf8 failed you will close convert_to_unicode_ anyway. So I assume you want both to succeed or consider them fail.  OK so I shouldn't say "both fail" but it's to explain that it's either both succeed or nothing so people don't have to try to deduce the intent.

Reply:
k, added a comment.